docs/intro.rst

```diff
@@ -0,0 +1,7 @@
+===============
+Getting started
+===============
+
+TODO: Write about how to include Resolwe in a Django project and explain
+settings parameters. Create an example Django project in the ``docs/example``
```

Comment:
Well, I started with this project because I needed a place for the example processors that I used in the docs. I suggest you start writing the getting started and you will see what kinds of example files you will need to explain how to kick-off resolwe and all its settings. Maybe you will see the test project sufficient, maybe not. We can always shuffle this files around.